vivisect/base.py

```diff
@@ -669,19 +677,28 @@ def _initFunction(self, funcva):
     #def _loadImportApi(self, apidict):
         #self._imp_api.update( apidict )
 
+    def getEndian(self):
+        return self.bigend
+
 
 #################################################################
 #
 #  setMeta key callbacks
 #
     def _mcb_Architecture(self, name, value):
+        archid = envi.getArchByName(value)
+        try:
+            # Some of the ENVI archs defined may not architecture modules that 
```

Comment:
Minor grammar nit: `Some of the ENVI archs defined may have architecture modules that`

Reply:
yeah.....
words.
```suggestion
            # Some of the ENVI archs defined may have architecture modules that 
```